staging/src/k8s.io/api/resource/v1beta1/types.go

```diff
@@ -31,6 +31,17 @@ const (
 	// Reserved for use by Kubernetes, DRA driver controllers must
 	// use their own finalizer.
 	Finalizer = "resource.kubernetes.io/delete-protection"
+	// ExtendedResourceClaimAnnotation is the annotation that generated special
```

Comment:
nit: ExtendedResourceClaimAnnotation is the annotation applied on the generated special ResourceClaim